### PR TITLE
Structure

### PR DESCRIPTION
**Pull request by codezix**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -5,8 +5,47 @@
...
+     </header>
+      <!-- create two colums -->
+ 
+      <!-- Add an about me message, social media icons in the first column -->
+     <div class="column" stlye="background-color: ;">
+            <!-- column 1 -->
+            <p>
+            <h2><strong>About me</strong></h2><br>
+            for the love of codes i build things.<br>
+            -----------------------------------------
```
Comment (GodwinEkuma) [inline about.html:36]: Is this supposed to be a comment?

File: contact.html [hunk 2]
```diff
@@ -5,8 +5,56 @@
...
+               <img src= "http://via.placeholder.com/20x20" alt= "logo">
+            </p>
+            <!-- contact adress -->
+            <p>
+               Address:
+               phone:
+            </p>
+            <!-- social mediaicons -->
+            <p>
+               <img src= "http://via.placeholder.com/20x20" alt= "logo">
```
Comment (GodwinEkuma) [inline contact.html]: each of img alt has logo as value. Add more context to it. Eg: `alt="facebook icon`.

File: contact.html [hunk 3]
```diff
@@ -5,8 +5,56 @@
...
+      <!-- Add a contact form in the second column -->
+    <form action="/action_page.php" method="POST" target="_blank">
+         Name: <br>
+         <input name="name" type="text" size="30"/><br><br>
+         Email: <br>
+         <input name="Email" type="text" size="30"/><br><br>
+         Message: <br>
+         <textarea name="message" cols="30" rows="5"></textarea>
+         <br>
+         <p><input type="submit" value="Send" /></p>
```
Comment (GodwinEkuma) [inline contact.html]: closing `/` is not necessary in an `input` element.

**Review by GodwinEkuma — CHANGES_REQUESTED** [review]

File: about.html (hunk 1 above)
Comment (codezix) [reply about.html:36]: They are suppose to be pseucode in form of comments. Explaining my html skeleton

File: contact.html (hunk 2 above)
Comment (codezix) [reply contact.html]: Thank you for your feedback, I have implemented it, and created a pull request for it

https://github.com/codezix/portfolio/pull/2

File: contact.html (hunk 3 above)
Comment (codezix) [reply contact.html]: Thank you for your feedback, I have implemented it, and created a pull request for it

https://github.com/codezix/portfolio/pull/2

**Review by GodwinEkuma — APPROVED** [review]
LGTM

**Pull request merged**